src/commands/play.ts

```diff
@@ -166,4 +169,9 @@ export default class implements Command {
       await player.play();
     }
   }
+
+  private async getPlaylistLimit(guildId: string) {
```

Comment:
I don't think you need a custom getter here as the column already defaults to 50.